andela/Ah-backend-guardians
# #162685682 Users can view their reading stats

#### What does this PR do?
Authenticated Users are able to view their reading stats


#### Description of Task to be completed?
These are the endpoints working
```
GET api/profiles/<username>/reading-stats/         get the reading stats of a particluar user
```

#### How should this be manually tested?
Run the following commands
` git fetch`
`git checkout ft-reading-stats-162685682`
`python3 manage.py makemigrations`
`python3 manage.py migrate`
`python3 manage.py runserver`

Open POSTMAN and test the endpoint above on various users.
The response body should look like this.
![image](https://user-images.githubusercontent.com/26769886/51912770-1eefe800-23e6-11e9-8f7e-4252e2aab352.png)


#### Any background context you want to provide?
The 'recent articles' key shows the last 5 articles read by a user.

A read is counted once a user hits the endpoint below:
```
GET api/articles/<slug>/
```

#### What are the relevant pivotal tracker stories?
#162685682
